static/app/views/discover/results.tsx

```diff
@@ -760,6 +760,7 @@ export class Results extends Component<Props, State> {
                       maxQueryLength={MAX_QUERY_LENGTH}
                       customMeasurements={contextValue?.customMeasurements ?? undefined}
                       dataset={eventView.dataset}
+                      includeTransactions={false}
```

Comment:
https://github.com/getsentry/sentry/blob/4c5824dacc10a7d7a5e3a36191d77e754f9d521b/src/sentry/tagstore/snuba/backend.py#L1199-L1200
includeTransactions sets dataset to Discover

Reply:
oh actually, i'm going to feature flag this, one sec